_data/betting.yml

```
@@ -21,6 +21,13 @@ websites:
       - totp
     doc: https://cloudbet.zendesk.com/hc/en-us/articles/223120088/
 
+  - name: Parimatch
+    url: https://www.parimatch.com/
+    img: parimatch.svg
+    tfa:
+      - totp
+    doc: https://air.parimatchgoal.com/en/faq/authentication/
```

```suggestion
    doc: https://faq.parimatch.com/en/account-access-and-security/
```
The doc link provided doesn't seem to work. Perhaps this one might be more suitable?